protos/merlin/transformer/spec/common.proto

```diff
@@ -23,6 +23,27 @@ message FromJson {
   ValueType valueType = 4;
 }
 
+message FromFile {
+  enum FileType {
+    CSV = 0;
+    PARQUET = 1;
+  }
+  FileType format = 1;
+  string uri = 2;
```

Comment:
Also since we use `uri` term here it should follow correct [convention](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#:~:text=The%20URI%20comprises%3A,%2C%20or%20hyphen%20(%20%2D%20).).
That means we can't use relative path (which we need for use case of accessing csv in model artifact).

As such, I think we should differentiate the field. So:

```
message FromFile {
  enum FileType {
    CSV = 0;
    PARQUET = 1;
  }

  // either uri or artifactPath must be populated
  string uri = 1; // e.g. gs://my-bucket/my-file.csv or file:///tmp/absolute/path/my-file.csv
  string artifactPath = 2; // e.g. path/my-file.csv --> standard transformer will translate it to file:///mnt/models/artifact/path/my-file.csv

  FileType format = 3;
  
  repeated Schema schema = 4;
}
```

Reply:
I prefer to stick to just 1 variable, say `filePath`. it is not that difficult to distinguish between a full and a relative path. However, if you give too many variables to the user, they will get confused and end up using wrongly.

Reply:
It's general purpose uri parsing https://go.dev/play/p/5PY9lpmEIkU